scripts/lint.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+SOURCES="piccolo_api tests"
+
+isort $SOURCES
+black $SOURCES
+flake8 $SOURCES
+# mypy $SOURCES
```

Comment:
Should we uncomment this?

Reply:
Ah, realised there are quite a few MyPy errors at the moment, so best to keep this commented.